src/makeInfinityAdder.js

```diff
@@ -6,7 +6,24 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let result;
+  let sumOfNum = 0;
+
+  const add = (num) => {
+    if (num === undefined) {
+      result = sumOfNum;
```

Comment:
you can declare the result here, no need to save it in closures

Reply:
Done!